docs/source/rbac/roles.md

```diff
@@ -0,0 +1,60 @@
+# Roles
+JupyterHub provides three **default roles** which are automatically loaded to the database at the startup:
+
+```{admonition} **Default roles**
+- **_user_** role carries single `all` scope that grants _least user access_ to perform only default user actions.
```

Comment:
> role carries single `all` scope

I think there's a word missing here, or there's some other grammatical error.
Also I was puzzled by the meaning of `all` until I got to the scopes page, would be nice to cross-reference the relevant section

Reply:
The user role has been rephrased and cross-referenced with the default user scope in the Scopes section.